Sources/StreamChat/Utils/InternetConnection/Error+InternetNotAvailable.swift

```diff
@@ -8,9 +8,23 @@ import Foundation
 /// to be extended as needed.
 private let offlineErrors: [(domain: String, errorCode: Int)] = [
     (NSURLErrorDomain, NSURLErrorNotConnectedToInternet),
+    (NSURLErrorDomain, NSURLErrorDataNotAllowed),
```

Comment:
The goal of this PR is to not show any alert if the user tries to connect offline. Each time I connect without the Internet, I receive this error. Strictly speaking, it's not an `offline` one but for some reason, URL session returns it instead of `NSURLErrorNotConnectedToInternet`.

Do you think we should include `NSURLErrorDataNotAllowed` to the offline list? @GetStream/ios-developers 

@GetStream/ios-developers

Reply:
Discussing it on Slack 🙂

Reply:
@polqf @nuno-vieira updated the logic to show a banner. Please 👀 when you have time

Reply:
Looks good 👌

Reply:
Hum 🤔 The only problem now is "Connecting..." is displayed always, even if the connection is good. Why did we make this change? Wasn't the goal to just discard any change for now. If the connection is offline, we don't do nothing for now

Reply:
Responded [here](https://github.com/GetStream/stream-chat-swift/pull/2011#discussion_r882474905)